pages/profile/read/views/profile.jsx

```diff
@@ -37,7 +38,9 @@ class Profile extends React.Component {
 
     return (
       <Fragment>
-        {title && <h2>{title}</h2>}
+        {title && <p><h1>{title}</h1></p>}
+
+        {estName && <h3>{estName}</h3>}
```

Comment:
Is this component re-used on the dashboard? If so then I don't think the profile name should be included here.